src/modules/shared/components/MediaCard/MediaCard.tsx

```diff
@@ -33,9 +46,74 @@ const MediaCard: FC<MediaCardProps> = ({
 	icon,
 	isKeyUser,
 	meemooIdentifier,
+	showLocallyAvailable = false,
+	link,
+	maintainerSlug,
 	hasTempAccess,
 }) => {
 	const { tText } = useTranslation();
+
+	const user = useSelector(selectUser);
+	const { mutateAsync: createVisitRequest } = useCreateVisitRequest();
+
+	const [isInfoModalOpen, setIsInfoModalOpen] = useState(false);
+	const [isRequestAccessBladeOpen, setIsRequestAccessBladeOpen] = useState(false);
+
+	const wrapInLink = (children: ReactNode) => {
+		if (showLocallyAvailable) {
+			return children;
+		}
+
+		if (link) {
+			return (
+				<Link key={id} href={link}>
+					<a className="u-text-no-decoration" aria-label={id}>
+						{children}
+					</a>
+				</Link>
+			);
+		}
+
+		return children;
```

Comment:
Je hoeft die `showLocallyAvailable` check niet uit te voeren aangezien je by default toch `children` returned. Als je wil vermijden dat je in de `link` if terecht komt indien `showLocallyAvailable` op true staat. Kan je je `link` check uitbreiden. 

```suggestion
		if (link && !showLocallyAvailable) {
			return (
				<Link key={id} href={link}>
					<a className="u-text-no-decoration" aria-label={id}>
						{children}
					</a>
				</Link>
			);
		}

		return children;
```